### PR TITLE
Fix check_process

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem
- *This app fails the CI because the check_process lacks admin, language, expiration, deletion*

## Solution
- *Fix check_process*
- *Small typo stuff*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Package_check results
---
*If you have access to [App Continuous Integration for packagers](https://yunohost.org/#/packaging_apps_ci) you can provide a link to the package_check results like below, replacing '-NUM-' in this link by the PR number and USERNAME by your username on the ci-apps-dev. Or you provide a screenshot or a pastebin of the results*

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/framaforms_ynh%20PR31%20(ericgaspar)/)  

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/framaforms_ynh%20PR31%20(yalh76)/)

### CODE REVIEW

File: scripts/restore [hunk 1]
```diff
@@ -75,7 +75,8 @@ ynh_script_progression --message="Restoring user rights..." --weight=1
 
 # Restore permissions on app files
 chown -R root: $final_path
-chmod 2775 "$final_path/sites/default/files"
+mkdir -p "$final_path/app/sites/default/files"
```
Comment (yalh76) [inline scripts/restore:78]: I don't think, creates that folder is required

File: manifest.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "en": "Create online webforms and surveys",
     "fr": "Créez des formulaires et questionnaires en ligne"
   },
-  "version": "1.0.1+200715~ynh2",
+  "version": "1.0.1~ynh1",
```
Comment (yalh76) [inline manifest.json:9]:
```suggestion
  "version": "1.0.1+200715~ynh3",
```

**Review by yalh76 — CHANGES_REQUESTED** [review]

File: scripts/restore (hunk 1 above)
Comment (ericgaspar) [reply scripts/restore:78]: If not created, the restore script fails :s

Comment (yalh76) [reply scripts/restore:78]: seems strange because you be present as restored during `ynh_restore_file --origin_path="$final_path"`

Comment (ericgaspar) [reply scripts/restore:78]: I am rerunning a test...

Comment (ericgaspar) [reply scripts/restore:78]: and ... you're right  o.O

**Review by zamentur — APPROVED** [review]

**Pull request merged**